### ISSUE TITLE
Figures are not included

### ISSUE THREAD

**Issue by jsumners** [fastify/website-metalsmith#225]
https://www.fastify.io/docs/latest/Encapsulation/

![image](https://user-images.githubusercontent.com/321201/105993679-ae334300-6074-11eb-8eb4-446d5cab6c22.png)

cc: @lmammino

**Comment by salesh** [comment]
I believe that url
```![Figure 1](./resources/encapsulation_context.svg)```  in https://github.com/fastify/fastify/blob/master/docs/Encapsulation.md should change to
```https://github.com/fastify/fastify/raw/master/docs/resources/encapsulation_context.svg```

Fastify website can't build with a link like that...

**Comment by lmammino** [comment]
That's a good one. We don't have any support for images (right now).

As  @salesh suggested, one workaround here might be to use the absolute image URL for the svg on the source file so that also the website ends up using an absolute URL.

```markdown
![Figure 1](https://github.com/fastify/fastify/raw/master/docs/resources/encapsulation_context.svg)
```

This is far from ideal. I think we should keep this issue and work something out to be able to process images in our website build.

**Comment by lmammino** [comment]
I gave it a quick try locally and I can confirm that the above works (as a workaround):

<img width="1445" alt="Screenshot 2021-01-28 at 09 32 51" src="https://user-images.githubusercontent.com/205629/106118297-d8a3ff80-614b-11eb-95ff-080f24dff9f5.png">

**Comment by lmammino** [comment]
Closed by #249
